modules/ROOT/pages/performance-and-impact.adoc

```diff
@@ -0,0 +1,19 @@
+= Anypoint Monitoring Performance and Impact
```

Comment:
Should the title be Anypoint Monitoring Performance Testing? Then the intro sentence  on line 3 could summarize "Review details about the performance impact and data retention with Anypoint Monitoring." or similar